geometry/spatial-operations/README.md

```diff
@@ -2,20 +2,20 @@
 
```

Comment:
For the title, it should be sentence case

Reply:
Thanks, updated to sentence case